packages/design-system/src/__generated__/figma-design-tokens.ts

```diff
@@ -320,6 +320,48 @@ export const typography = {
     textDecoration: "underline",
     textIndent: "0px",
   },
+  regularLink: {
+    fontFamily:
+      "Inter Variable, InterVariable, Inter, -apple-system, system-ui, sans-serif",
+    fontWeight: 400,
+    fontSize: "12px",
+    lineHeight: "16px",
+    letterSpacing: "0.005em",
+    textTransform: "none",
+    textDecoration: "underline",
+    textIndent: "0px",
+  },
+  labelLink: {
+    fontFamily:
+      "Inter Variable, InterVariable, Inter, -apple-system, system-ui, sans-serif",
+    fontWeight: 500,
+    fontSize: "12px",
+    lineHeight: "16px",
+    letterSpacing: "0.005em",
+    textTransform: "none",
+    textDecoration: "underline",
+    textIndent: "0px",
+  },
+  monoBoldLink: {
+    fontFamily: "Roboto Mono, RobotoMono, menlo, monospace",
+    fontWeight: 500,
```

Comment:
mono bold and mono are same font weight, is it intentional?

Reply:
Why do we need monoBold if we have the same value

Reply:
That was a mistake. Ijust fixed it in figma and pushed to github. LMK if I need to make a PR for that update

Reply:
yes need a pr but also need to clarify everything about the link colors, I tried to systematize and make them usable in code and it was a bad experience, nothing is clear:

seems like we need

Colors
- main (hover & visited)
- contrast (hover & visited)
- subtle (hover & visited)
- moreSubtle (hover & visited)

Variants
- regular
- label
- mono

Reply:
Unless we completely don't care about visited, then each color has to have default and hover color

Reply:
Actually I don't think we need visited colors at all

Reply:
Yea the color tokens could use a re-organization if you think its worth investing in that.

All those colors that you listed are tokens, they are just organized so as not to disrupt the greater system. Link hovers are not just called "hovers" becuase they are specifically for text links and nothing else. Regular text does change color on hover.

Yes you can group the link hovers with their respective normal text colors, but I made the decision to group them separately and together instead.

Current: foreground.link-hover.main   - groups link-hovers within foreground
Possible: foreground.main.link-hover - puts each link-hover within its reference color

Reply:
I wouldn't wawnt to toouch all colors right now but I want to understand the links and I don't

Reply:
This is what I see we have in total for links:

  foregroundLink: "#006adc",
  foregroundLinkVisited: "#793aaf",
  foregroundLinkNotVisited: "#006adc",
  foregroundLinkHoverMain: "#656869",
  foregroundLinkHoverTextSubtle: "#656869",
  foregroundLinkHoverTextMoreSubtle: "#656869",
  foregroundLinkHoverContrast: "#adb1b4",

Reply:
These I am unfamiliar with. I think @markxtang must have added them. I would say they should be deleted, but Mark should confirm first that it will not break a design that he made.

> foregroundLink: "#006adc",
> foregroundLinkVisited: "#793aaf",
> foregroundLinkNotVisited: "#006adc",

Reply:
is there no way to check if they are used anywhere in figma? 

I mean if there is "bus factor", what do we do?

Reply:
in code these 2 only used in 2 places: unsupported browser alert and publish dialog

foregroundLink: "#006adc",
foregroundLinkVisited: "#793aaf",

Reply:
There's not a native way, but I found a plugin that finds where styles are used. I confirmed none of these tokens are used in any designs, so I removed all 3.

I don't know why these are used in code but not in the design. You should check each of those instances to see if they need to be replaced with one of the new link-hover colors, or just leave it the same color but not connected to a token.